.changeset/silly-bananas-peel.md

```diff
@@ -0,0 +1,12 @@
+---
+"@itwin/itwinui-css": major
+---
+
+Replaces the size classes for `iui-avatar` with a new `data-iui-size` attribute.
+
+```diff
+-<div class="iui-avatar iui-small">
++<div class="iui-avatar" data-iui-size="small">
```

Comment:
nit: Spaces makes it easier to read.


```suggestion
- <div class="iui-avatar iui-small">
+ <div class="iui-avatar" data-iui-size="small">
```

Same for the other changeset.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1935/commits/82d4feb21e6ac25c68efaf2cbeaa8b59d1ba02a5)